ganymede/models/stahl/stahl.yml

```diff
@@ -5,12 +5,43 @@ models:
   - name: dim_store
     description: |
       A dimension table containing all relevant attributes of stores. 
+  - name: dim_census_division
+    description: |
+      A dimension table containing all relevant attributes of a Canadian census division.
     columns:
+      - name: id
+        description: |
+          The unique identifier for the census division directly obtained from the `cduid`
+          in the statcan source
       - name: census_division_name
         description: |
           This string specifies the census division each store resides in, where each census division is a StatCan-designated geographic subregion.
           The PostGIS extension is used to locate each store within a census division, and this not null test verifies that each store has in fact been
           associated with a census division. The set of all census divisions form a mutually disjoint cover of all of Canada, and hence each store
           should be located within exactly one census division.   
+      - name: census_division_type
+        description: |
+          The census division type comes from the Canadian government census definitions.
+          This current resource contains only CDR (Census Divisions / de recensement) used
+          to establish provincial law and facilitate regional planning. Other valid division
+          types include TER (territory), CSD (Census Subdivision), MB (Management Board) etc.
+
+          More information can be found here:
+          https://www12.statcan.gc.ca/census-recensement/2021/ref/dict/az/definition-eng.cfm?ID=geo008
+        tests:
+          - accepted_values:
+              values: ['CDR']
+      - name: land_area
+        description: |
+          Land area is the area in square kilometres of the land-based portions
+          of standard geographic areas. The land area measurements are unofficial
+          and are provided for the sole purpose of calculating population density.
+      - name: region_code
+        description: |
+          The two letter international standard denomination for the country's subdivisions established by the ISO-3166-2 code.
+          This can be a province, territory, state etc. Uses `raw_region_to_province` macro for parsing of the raw region code.
+          If this accepted values test breaks, it means that there was a new region code detected and the logic should be updated
+          to handle the new values if possible.
         tests:
-          - not_null
```

Comment:
Why remove this test?

Reply:
Truthfully, it's because the resource is immutable and I already tested this behavior. Having this test creates a compiled sql statement as an artifact that can potentially be executed against the db and we should try to prevent unnecessary load on the db now that we're scaling input but not increasing database spend.

Reply:
Altho the original purpose of this was as a test for the postgis functionality--i.e. if the test passes then we can be assured that all stores have been associated with a census division. Isn't there the possibility that new stores are added to aethervest, in which case we would want to be able to execute this test again?

Reply:
Oh... that's a good point.